Index: src/main/kotlin/de/rumeotech/minelin/Minelin.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/de/rumeotech/minelin/Minelin.kt b/src/main/kotlin/de/rumeotech/minelin/Minelin.kt
new file mode 100644
--- /dev/null	(date 1666470272824)
+++ b/src/main/kotlin/de/rumeotech/minelin/Minelin.kt	(date 1666470272824)
@@ -0,0 +1,10 @@
+package de.rumeotech.minelin
+
+object Minelin {
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        println("Hello World!")
+    }
+
+}
\ No newline at end of file
